modules/tensor_mechanics/include/materials/StressUpdateBase.h

```diff
@@ -19,6 +19,24 @@
 // Forward declaration
 class StressUpdateBase;
 
+/**
+ * TangentCalculationMethod is an enum that determines the calculation method for the tangent
+ * operator
+ * ELASTIC uses the elasticity tensor as the tangent operator: J = C. If models are elastic, the
```

Comment:
I'd replace the second sentence with this: "The elasticity tensor does not need to be provided by the StressUpdateBase models in this case".